tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/CatalogueSolutionsControllerTests.cs

```diff
@@ -332,6 +332,217 @@ public async Task CreateOrderItemAsync_CreateOrderItemService_CalledOnce()
                     x.CreateAsync(It.IsNotNull<CreateOrderItemRequest>()), Times.Once);
         }
 
+        [Test]
+        public void UpdateOrderItemAsync_NullModel_ThrowsArgumentNullException()
+        {
+            static async Task TestAsync()
```

Comment:
I'm not a fan of these static methods, as if anything in the test setup was to throw the exception, the test would pass, not only if the function we're wanting to call did. Prefer to do everything in the test as normal, then do

Assert.ThrowsAsync<ArgumentNullException(async () => context.Controller.UpdateOrderItemAsync(orderId, orderItemId, null)

Plus it lets us avoid nested functions which is always a bonus. Won't request a change, one for a code standards meeting though I think

Reply:
Yeah one for the next coding standards meeting then. Happy to go with the consensus.